### PR TITLE
Add fivetran quickstart

### PR DESCRIPTION
**Pull request by glavanr**
# Summary
Adding Fivetran quick start dashboard and alert conditions.

## Pre merge checklist

- [ x] Did you check you NRQL syntax? - Does it work?
- [ x] Did you include a Data source and Documentation reference?
- [ x] Are all documentation links publicly accessible?
- [ x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ x] Did you check your descriptive content for spelling and grammar errors?
- [ x] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [x ] Does the PR contain a screenshot for each of your dashboards?
- [x ] Do your screenshots show data?
- [x ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ x] Did you check that your alerts actually work?

### CODE REVIEW

File: alert-policies/fivetran/ConnectionFailure.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: FIVETRAN_CONNECTION_FAILURE
+
+description: |+
+  This alert is trigged for any Fivetran jobs failed by a connection_failure event.
```
Comment (sjyothi54) [inline alert-policies/fivetran/ConnectionFailure.yml:4]:
```suggestion
 This alert is triggered in the event of a connection failure, indicating the failure of any Fivetran jobs.
```

File: alert-policies/fivetran/FailureWithTask.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+name: FIVETRAN_CONNECTION_FAILURE
```
Comment (sjyothi54) [inline alert-policies/fivetran/FailureWithTask.yml:1]:
```suggestion
name: Task failure
```

File: alert-policies/fivetran/FailureWithTask.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+description: |+
+  This alert is trigged for any Fivetran jobs with an unsuccessful sync_end event.
```
Comment (sjyothi54) [inline alert-policies/fivetran/FailureWithTask.yml:4]:
```suggestion
  This alert is triggered in the event of a connection failure, indicating the failure of any Fivetran task.
```

File: data-sources/fivetran/config.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+id: fivetran
+displayName: Fivetran
+description: |
+  Fivetran is a SaaS solution that provides automated data movement.
```
Comment (sjyothi54) [inline data-sources/fivetran/config.yml:4]:
```suggestion
  Fivetran is a cloud-based data integration platform that helps organizations streamline the process of extracting, loading, and transforming (ELT) data from various sources into a centralized data warehouse.
```

File: quickstarts/fivetran/logo.svg
Comment (sjyothi54) [inline quickstarts/fivetran/logo.svg:1]: please use Fivetran logo instead of New Relic logo

File: quickstarts/fivetran/config.yml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  - fivetran
+
+# Reference to dashboards to be included in this quickstart
+dashboards:
+  - fivetran
+
+documentation:
+  - name: Fivetran New Relic Setup Guide
+    url: https://fivetran.com/docs/logs/external-logs/new-relic/setup-guide
+    description: Setup exteral logging with New Relic
```
Comment (sjyothi54) [inline quickstarts/fivetran/config.yml:31]:
```suggestion
    description: Setup external logging with New Relic
```

File: quickstarts/fivetran/config.yml [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+  - pipeline
+
+# Reference to alert policies to be included in this quickstart
+alertPolicies:
+  - fivetran
+
+# Reference to dashboards to be included in this quickstart
+dashboards:
+  - fivetran
+
```
Comment (sjyothi54) [inline quickstarts/fivetran/config.yml:27]:
```suggestion
dataSourceIds:
 - fivetran
```

File: data-sources/fivetran/config.yml [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+icon: fivetran-logo.png
+install:
+  primary:
+    link:
+      url: https://www.fivetran.com/
```
Comment (sjyothi54) [inline data-sources/fivetran/config.yml:9]:
```suggestion
      url: https://fivetran.com/docs/logs/external-logs/new-relic/setup-guide
```

**Review by sjyothi54 — CHANGES_REQUESTED** [review]
Please make this changes and could you please let us know why you have added 3 tabs i.e, production, eu_production and staging in the fivetran dashboard because in the customer point of view, they wont have these options.

File: quickstarts/fivetran/config.yml [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+level: Community
+
+authors:
+  - Michael Young
+
+keywords:
+  - fivetran
+  - connector
+  - pipeline
+
```
Comment (sjyothi54) [inline quickstarts/fivetran/config.yml:19]:
```suggestion
icon: logo.svg
```
Update the logo file name here

File: dashboards/fivetran/fivetran-dashboard.png
Comment (brammerl) [inline dashboards/fivetran/fivetran-dashboard.png:1]: Let's replace this with a photo that doesn't have the tabs. It makes it look like this quickstart will create multiple dashboards under multiple tabs which I'm assuming isn't the case. Perhaps take a screenshot of just one tab's view.

**Review by brammerl — CHANGES_REQUESTED** [review]

**Comment by glavanr** [comment]
thanks so much I will update these changes

**Comment by glavanr** [comment]
Changes have been made based on suggested edits.

File: quickstarts/fivetran/config.yml [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+level: Community
+
+authors:
+  - Michael Young
+
+keywords:
+  - fivetran
+  - connector
+  - pipeline
+icon: logo.svg
```
Comment (sjyothi54) [inline quickstarts/fivetran/config.yml:19]:
```suggestion
icon: logo.jpeg
```

**Review by sjyothi54 — CHANGES_REQUESTED** [review]
Please make the required changes and rest everything is fine.

**Comment by glavanr** [comment]
> Please make this changes and could you please let us know why you have added 3 tabs i.e, production, eu_production and staging in the fivetran dashboard because in the customer point of view, they wont have these options.

I took a new screenshot without the tabs.

File: alert-policies/fivetran/FailureWithTask.yml [hunk 10]
```diff
@@ -0,0 +1,28 @@
+name: Task failure
+
+description: |+
+  This alert is triggered in the event of a connection failure, indicating the failure of any Fivetran task.
+  
+type: STATIC
+
+nrql:
+  query: ""SELECT count(*) from Log where event = 'sync_end' and data.status ='FAILURE_WITH_TASK'"
```
Comment (brammerl) [inline alert-policies/fivetran/FailureWithTask.yml:9]:
```suggestion
  query: "SELECT count(*) from Log where event = 'sync_end' and data.status ='FAILURE_WITH_TASK'"
```

File: alert-policies/fivetran/ConnectionFailure.yml [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+  query: "SELECT count(*) from Log where event = 'connection_failure'"
+
+# Function used to aggregate the NRQL query value(s) for comparison to the terms.threshold (Default: SINGLE_VALUE)
+valueFunction: SINGLE_VALUE 
+
+# List of Critical and Warning thresholds for the condition
+terms:
+  - priority: CRITICAL
+    # Operator used to compare against the threshold.
+    operator: ABOVE_OR_EQUALS
```
Comment (brammerl) [inline alert-policies/fivetran/ConnectionFailure.yml:18]: `operator` must be one of: ABOVE, BELOW, EQUALS

File: alert-policies/fivetran/FailureWithTask.yml [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+  query: "SELECT count(*) from Log where event = 'sync_end' and data.status ='FAILURE_WITH_TASK'"
+
+# Function used to aggregate the NRQL query value(s) for comparison to the terms.threshold (Default: SINGLE_VALUE)
+valueFunction: SINGLE_VALUE 
+
+# List of Critical and Warning thresholds for the condition
+terms:
+  - priority: CRITICAL
+    # Operator used to compare against the threshold.
+    operator: ABOVE_OR_EQUALS
```
Comment (brammerl) [inline alert-policies/fivetran/FailureWithTask.yml:18]: `operator` must be one of: ABOVE, BELOW, EQUALS

File: quickstarts/fivetran/config.yml [hunk 13]
```diff
@@ -0,0 +1,32 @@
+slug: fivetran-integration
```
Comment (brammerl) [inline quickstarts/fivetran/config.yml:1]: [nit] this could simply be `fivetran` since there aren't any other quickstarts under this name.

**Review by brammerl — APPROVED** [review]
LGTM just a small [nit] I wanted to get your input on!

File: quickstarts/fivetran/config.yml (hunk 13 above)
Comment (glavanr) [reply quickstarts/fivetran/config.yml:1]: Not a problem.

**Comment by brammerl** [comment]
@sjyothi54 are you able to approve this pr? It seems the changes you requested have been made and this is a time sensitive quickstart.

**Review by brammerl — APPROVED** [review]

**Review by sjyothi54 — APPROVED** [review]
LGTM

**Pull request merged**